test/functional/pdo_sqlsrv/MsCommon_mid-refactor.inc

```diff
@@ -81,7 +81,7 @@ function getDSN($sqlsrvserver, $database, $keywords = '', $disableCE = false)
         $dsn .= "ColumnEncryption=Enabled;";
```

Comment:
For sqlsrv tests, this part is like `if (isColEncrypted() && !$disableCE)`... should we change in sqlsrv too?

Reply:
isColEncrypted is true only if `$keystore != 'none'` and `$dataEncrypted == false`, but I think we want to connect with `ColumnEncryption` enabled as long as `keystore` is not none. `dataEncrypted` only applies to table creation and row insertion. So yes, please change this in the SQLSRV side. But be aware that some tests will begin to fail because a lot of AE features and limitations are base on ColumnEncrypted connection.